test/assert.js

```diff
@@ -1454,6 +1455,14 @@ describe('assert', function () {
     err(function () {
       assert.property(undefined, 'a', 'blah');
     }, "blah: Target cannot be null or undefined.");
+
+    err(function () {
+      assert.propertyVal(dummyObj, 'a', '2', 'blah');
+    }, "blah: expected { a: '1' } to have property 'a' of '2', but got '1'");
+
+    err(function () {
+      assert.nestedProperty({a:1}, {'a':'1'}, 'blah');
+    }, 'blah: the argument to property must be a string when using nested syntax');
```

Comment:
One more test should be added here to the `assert` interface tests in order to verify the error message "the argument to property must be a string, number, or symbol". This test can closely resemble the `expect({a:1}).to.have.property(null);` test that you added on the `expect` interface, except using the `assert` interface instead of `expect`. (And using the `err` helper function, just like you've done in the previous two tests on the `assert` interface).